src/ios/CDVWKInAppBrowser.m

```diff
@@ -930,34 +940,43 @@ - (void)createViews
     self.addressLabel.textAlignment = NSTextAlignmentLeft;
     self.addressLabel.textColor = [UIColor colorWithWhite:1.000 alpha:1.000];
     self.addressLabel.userInteractionEnabled = NO;
-    
-    NSString* frontArrowString = NSLocalizedString(@"►", nil); // create arrow from Unicode char
```

Comment:
I actually wonder if instead of the pngs, we should go back to this, but maybe using a `<` and a `>` to keep it smaller. Or maybe there's a unicode character more like what we want it to be.

Reply:
I tried `<` and `>` and searched all over for unicode characters that would look good, but they all looked kinda funky. They were either too pointy or the alignment was off.